### PR TITLE
fix(ardour-git): move symlinks to `postinst` & fix various install issues

### PR DESCRIPTION
**Pull request by echometerain**

### CODE REVIEW

**Review by Henryws — CHANGES_REQUESTED** [review]
You can't use `$STOWDIR` in `postinst` and `removescript`. Once you are in those functions, you are not dealing with symlinks.

Example:
```bash
sudo ln -sf "$STOWDIR/$name/opt/bla" "$STOWDIR/$name/usr/bin/bla"

# should be

sudo ln -sf "/opt/bla" "/usr/bin/bla"
```

File: packages/ardour-git/ardour-git.pacscript [hunk 1]
```diff
@@ -3,15 +3,15 @@ maintainer="echometerain <echometer@disroot.org>"
...
 
 build_depends="libgio2.0-cil-dev python3 libqm-dsp-dev libcairo2-dev libglib2.0-dev libglibmm-2.4-dev libgtkmm-2.4-dev libx11-dev libxml2-dev libsoundtouch-dev libtag1-dev libatkmm-1.6-dev libaubio-dev libboost-all-dev libcairomm-1.0-dev libcppunit-dev libdbus-1-dev doxygen libfftw3-dev libflac-dev libfontconfig-dev libfreetype6-dev libgdk-pixbuf2.0-0 graphviz libgtk2.0-dev libhidapi-dev itstool libjack-jackd2-dev libarchive-dev liblo-dev libltc-dev libogg-dev libpulse-dev libsamplerate0-dev libsndfile1-dev libusb-1.0-0-dev libwebsockets-dev liblilv-dev lv2-dev libpango1.0-dev libpangomm-1.4-dev libreadline-dev librubberband-dev libserd-dev libsord-dev libsratom-dev libsuil-dev vamp-plugin-sdk libfluidsynth-dev liblrdf0-dev libraptor2-dev libcurl4-gnutls-dev"
 
 optdepends=("xjadeo: video monitoring"
    "harvid: video timeline"
    "linuxaudio-new-session-manager: for session management")
-gives="ladspa-host, lv2-host, vst-host, vst3-host, ardour, ardour-data"
-breaks="ardour, ardour-data"
+provides="ladspa-host lv2-host vst-host vst3-host ardour ardour-data"
```
Comment (Henryws) [inline packages/ardour-git/ardour-git.pacscript:13]: Make this an array

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
Add python as a build dependency

**Comment by Henryws** [comment]
Still fails.

**Comment by Henryws** [comment]
I’m gonna test manually when I get home.

**Comment by Henryws** [comment]
Ok turns out the CI was installing the repo one... New issue now.

**Review by cat-master21 — CHANGES_REQUESTED** [review]
yes this may add a extra dependency but it solves the issue of being able to work on multiple systems. I wish I could do it if the package provides it but I don't know if pacstall supports it.

**Comment by Henryws** [comment]
Try adding [libasound2](https://packages.debian.org/libasound2) to the build deps list.

**Comment by Henryws** [comment]
Why can’t debian detect the alsa package?

**Review by Henryws — APPROVED** [review]

**Pull request merged**